src/Fetch.js

```diff
@@ -367,27 +372,35 @@ function fetchXHR(fetch, onsuccess, onerror, onprogress, onreadystatechange) {
     }
   };
   xhr.onerror = (e) => {
-    saveResponse(fetchAttrLoadToMemory);
+    // check if xhr was aborted by user and don't try to call back
+    if (!(id in Fetch.xhrs)) { 
+      return;
+    }
     var status = xhr.status; // XXX TODO: Overwriting xhr.status doesn't work here, so don't override anywhere else either.
```

Comment:
Looks like this variable is no longer needed?   (I can't really see what the point of this line was?)